core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/AuditingSettingsDecoder.scala

```diff
@@ -34,21 +39,91 @@ import scala.util.{Failure, Success, Try}
 
 object AuditingSettingsDecoder extends Logging {
 
-  lazy val instance: Decoder[Option[AuditingTool.Settings]] =
-    Decoder.instance { c =>
-      whenEnabled(c) {
-        for {
-          auditIndexTemplate <- decodeOptionalSetting[RorAuditIndexTemplate](c)("index_template", fallbackKey = "audit_index_template")
-          customAuditSerializer <- decodeOptionalSetting[AuditLogSerializer](c)("serializer", fallbackKey = "audit_serializer")
-          remoteAuditCluster <- decodeOptionalSetting[AuditCluster.RemoteAuditCluster](c)("cluster", fallbackKey = "audit_cluster")
-        } yield AuditingTool.Settings(
-          auditIndexTemplate.getOrElse(RorAuditIndexTemplate.default),
-          customAuditSerializer.getOrElse(new DefaultAuditLogSerializer),
-          remoteAuditCluster.getOrElse(AuditCluster.LocalAuditCluster)
-        )
+  lazy val instance: Decoder[Option[AuditingTool.Settings]] = {
+    for {
```

Comment:
why not `auditSettingsDecoder.orElse(DeprecatedAuditSeettingsDecoder.instance)`?